iconic/UserProfile/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import include, path
+
+from . import views
+
+urlpatterns = [
+    path('user/wallet/', views.UserWalletView.as_view(),
+         name='amount-wallet')
+]

```

Comment:
change path to "wallet/" and add "user/" path to iconic/urls.py like 
    "path('users/', include('UserProfile.urls'))"